qadence/backends/pulser/backend.py

```diff
@@ -72,27 +67,14 @@ def make_sequence(circ: QuantumCircuit, config: Configuration) -> Sequence:
     else:
         raise ValueError("Specified device is not supported.")
 
-    max_amp = device.channels["rydberg_global"].max_amp
-
-    if config.spacing is not None:
-        spacing = config.spacing
-    elif max_amp is not None:
-        # TODO: Fix this more consistently so both pulser and
-        # pyqtorch get the spacing from the same place
-        # Ideal spacing for entanglement gate
-        # since Pulser's QutipEmulator doesn't allow simulation of sequences
-        # with total duration < 4ns
-        spacing = WEAK_COUPLING_CONST * device.rydberg_blockade_radius(max_amp)  # type: ignore
-    else:
-        spacing = DEFAULT_SPACING
```

Comment:
awesome

Reply:
I would like to ask @kaosmicadei about it but he is on vacations 😅

Reply:
Either way I've sent him a message about it to discuss when he's back.